app/models.py

```diff
@@ -200,6 +209,6 @@ def __repr__(self):
 class VoteInformation(db.Model):
     voter_apikey = db.Column(db.String, db.ForeignKey('key.apikey'), primary_key=True)
     resource_id = db.Column(db.Integer, db.ForeignKey('resource.id'), primary_key=True)
-    current_direction = db.Column(db.String, nullable=False)
+    current_direction = db.Column(db.String, nullable=True)
```

Comment:
I think this is going to require a db migration, do you know how to generate one?

Reply:
Does running `make migrate` take care of this? Does that remove all the existing vote information? Asking since due to a change in `update_votes`, there could some unexpected bugs

Reply:
No, `make migrate` runs a migration that's been created but hasn't been applied to your local db. It's what developers will have to do for their local environments after we merge. 

Though I will say, we should probably rename that as `make upgrade` and get rid of the migrate command in there because I don't think it does what I thought it did when I wrote that into the Dockerfile.

You'll want to name your migration, so you'll do something like `docker-compose run resources-api flask db migrate -m "include vote direction"` and then you'll need to check it to ensure that everything worked correctly in the auto generated file, then upgrade your database with `docker-compose run resources-api flask db upgrade` and if it successfully upgrades the database, then you can commit that file.

See also https://flask-migrate.readthedocs.io/en/latest/#example

Reply:
I ran the migration command but saw the message below. 

<img width="1174" alt="Screen Shot 2020-11-05 at 9 21 49 PM" src="https://user-images.githubusercontent.com/17136024/98318775-49080e00-1fad-11eb-9c18-6344d5269da9.png">

Reply:
Hmm, I saw something different when I ran it. Maybe I'll just go ahead and generate the file and push it up to your branch